spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/CompositePropagationFactory.java

```diff
@@ -70,10 +69,19 @@ public boolean requires128BitTraceId() {
 		return this.injectors.requires128BitTraceId() || this.extractors.requires128BitTraceId();
 	}
 
+	/**
+	 * @deprecated end users and instrumentation should never call this, and instead
+	 * use {@link #get()}. This only remains to avoid rev-lock upgrading to Brave 6.
+	 */
+	@Deprecated
```

Comment:
There's no point in deprecating this as it's package-private API for which the javadoc is not published.